Source/OxyPlot.Avalonia/OxyPlotModule.cs

```diff
@@ -8,10 +9,9 @@ namespace OxyPlot.Avalonia
 
     public static class OxyPlotModule
     {
-        public static void Initialize()
+        public static void EnsureLoaded()
         {
-            AvaloniaLocator.Current.GetService<IGlobalStyles>().Styles.AddRange(new Themes.Default());
+
         }
```

Comment:
Unused `using`s. Can we delete this class?

Reply:
obsolete